Makefile

```diff
@@ -31,12 +31,12 @@ build:
 build-debug:
 	$(MAKE) build BUILD_OPTS='-gcflags=all="-N -l"'
 
-# Run the CLO locally. Requires a cluster login, sets up all other pre-requisites.
-# Can be run multiple times without any undeploy/uninstall steps.
+# Run the CLO locally - see HACKING.md
 RUN_CMD?=go run
 run:
-	$(MAKE) deploy-elasticsearch-operator
 	@oc get ns $(NAMESPACE) &> /dev/null || oc create ns $(NAMESPACE)
+	@oc delete -n $(NAMESPACE) all --all
```

Comment:
IIRC `oc delete all` will not delete CRDs or instances of those. Same applies for ClusterRoles and ClusterRoleBindings.

Reply:
Correct - it's sufficient for re-starting a `make run` CLO but it won't clean up leftovers from aborted tests. I've found I have to clean up a whole list of service addresses and other things. See the `test-cleanup` target for a start at doing a proper house-cleaning. I haven't documented it yet because it doesn't really work and I'm not sure it's the best approach. The tests have code to do their own cleanup, but it appears not to always work if tests are interrupted. I think the tests themselves shoudl do a pre-clean to prepare their own preconditions rather than assuming that previous tests left things clean. Anyhow - all that will need to be the subject of another PR.